media-gfx/yafaray/yafaray-3.3.0.ebuild

```diff
@@ -0,0 +1,123 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{5,6} )
+inherit cmake-utils python-r1 python-utils-r1
+
+DESCRIPTION="A free open-source montecarlo raytracing engine."
+HOMEPAGE="http://www.yafaray.org"
+SRC_URI="https://github.com/YafaRay/Core/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/YafaRay/Blender-Exporter/archive/v${PV}.tar.gz -> ${PN}-blender-exporter-${PV}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+blender
+	debug
+	+fastmath
+	+fasttrig
+	jpeg
+	opencv
+	openexr
+	png
+	+python
+	qt4
```

Comment:
No more qt4 for new ebuilds. Was this option not ported to Qt5?

Reply:
Noted. The CMakeLists.txt is looking specifically for 4.4.0 and it links against Qt4. I'm not exactly sure what the Qt support is for as it builds a .so, not an executable, and the main binary doesn't have a GUI option. I'll ask upstream about Qt5 and see if I can track down more information on the feature.